### ISSUE TITLE
When will the full dataset be released?

### ISSUE THREAD

**Issue by Jasper0420** [HuajianUP/360VOT#2]
Thank you for your excellent work, I now see that only the test set can be downloaded, I would like to know roughly when the training set will be available for download?

**Comment by Jasper0420** [comment]
Thanks again for your creative work, looking forward to it!
